build.gradle

```diff
@@ -312,7 +312,13 @@ dependencies {
     }
   implementation group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
 
-  implementation group: 'com.github.hmcts.java-logging', name: 'logging', version: versions.reformLogging
+  implementation group: 'com.github.hmcts.java-logging', name: 'logging', version : versions.reformLogging
```

Comment:
remove space  after version. should be "version:"

Reply:
removed extra space